packages/graphql/src/translate/translate-update.ts

```diff
@@ -108,52 +106,75 @@ function translateUpdate({ node, context }: { node: Node; context: Context }): [
             }
 
             if (relationField.interface) {
```

Comment:
These functions were re-ordered and placed in the `cypher` array because the order by which withProjector.nextWith is called matters. This also enhances readability.